addons/editor-devtools/DevTools.js

```diff
@@ -401,6 +430,18 @@ export default class DevTools {
     }
   }
 
+  _canPaste(workspace) {
+    // Don't paste if the mouse is outside the workspace SVG group
+    const group = workspace.svgGroup_;
+    const bounds = group.getBoundingClientRect();
```

Comment:
The group's bounds include blocks that are scrolled out of view:

![image](https://github.com/user-attachments/assets/a1b57386-8bda-494e-8b64-9e5365e13faf)

You can get the actual bounds from the `.blocklySvg`:

![image](https://github.com/user-attachments/assets/adcd343f-7556-4a4e-9fd2-11ecb26f3a19)

```suggestion
    // Don't paste if the mouse is outside the workspace SVG
    const svg = document.querySelector(".blocklySvg");
    const bounds = svg.getBoundingClientRect();
```

Reply:
fixable I guess